src/uct/ib/rc/base/rc_iface.c

```diff
@@ -14,7 +14,7 @@
 
 
 ucs_config_field_t uct_rc_iface_config_table[] = {
-  {"IB_", "RX_INLINE=64;RX_QUEUE_LEN=4095", NULL,
+  {"IB_", "RX_INLINE=64;RX_QUEUE_LEN=256", NULL,
```

Comment:
FC window is limited by RX_QUEUE_LEN ([here](https://github.com/openucx/ucx/blob/master/src/uct/ib/rc/base/rc_iface.c#L445)).
However FC may not work efficiently, because the whole RX pool is available for single sender.
BTW, It is configurable value, why wouldn't the one who needs lower memory consumption change it via en var?